lib/perl/Genome/Model/Tools/Capture/FormatSnvs.pm

```diff
@@ -60,10 +63,19 @@ sub execute {                               # replace with real execution logic.
 
 	## Get required parameters ##
 	my $variants_file = $self->variants_file;
-	my $output_file = $self->output_file;
+	my $output_file;
+	if($self->output_file) {
+		$output_file = $self->output_file;
+	} elsif($self->outdir) {
+		$output_file = File::Spec->join(
+		$self->outdir,
+		basename($variants_file) . ".formatted");
+	} else {
+		die $self->error_message("Specify variants file or outdir.");
```

Comment:
Should this error message refer to the output_file rather than the variants file?